pkg/api/types.go

```diff
@@ -3303,6 +3305,24 @@ const (
 	NodeInternalDNS NodeAddressType = "InternalDNS"
 )
 
+// AllNodeAddressTypes exports all existing node address types
+var AllNodeAddressTypes = NodeAddressTypes{
```

Comment:
do we enumerate all constants of a type elsewhere? I would move this into the `cmd/kube-apiserver/app/options/validation.go`.

Reply:
Yes,  Reconciler types.
Moved